packages/components/psammead-most-read/src/index.stories.jsx

```diff
@@ -7,31 +7,41 @@ import Timestamp from '@bbc/psammead-timestamp';
 import MostReadList from './List';
 import MostReadTitle from './Title';
 import { MostReadLink, MostReadRank } from './Item';
-import { items, itemsRTL } from './testHelpers/fixtureData';
 import MostRead from './index';
 
 const arabicServiceDecorator = withServicesKnob({
   defaultService: 'arabic',
 });
 
-const bengaliServiceDecorator = withServicesKnob({
-  defaultService: 'bengali',
+const newsServiceDecorator = withServicesKnob({
+  defaultService: 'news',
 });
 
-const burmeseServiceDecorator = withServicesKnob({
-  defaultService: 'burmese',
+const BASE_PATH = 'https://www.bbc.com';
+const getItem = (articlePath, text) => ({
+  title: text,
+  href: `${BASE_PATH}${articlePath}`,
 });
 
-const newsServiceDecorator = withServicesKnob({
-  defaultService: 'news',
-});
+const getItems = (articlePath, text) =>
+  Array(10).fill(getItem(articlePath, text));
 
-const renderLTRList = ({ service, script }) => (
-  <MostReadList items={items} service={service} script={script} dir="ltr" />
+const renderLTRList = ({ service, script, articlePath, text, dir }) => (
+  <MostReadList
+    items={getItems(articlePath, text)}
+    service={service}
+    script={script}
+    dir={dir}
+  />
 );
 
-const renderTRLList = ({ service, script }) => (
-  <MostReadList items={itemsRTL} service={service} script={script} dir="rtl" />
+const renderRTLList = ({ service, script, articlePath, text, dir }) => (
+  <MostReadList
+    items={getItems(articlePath, text)}
+    service={service}
+    script={script}
+    dir={dir}
+  />
```

Comment:
from what is see, these are now identical, so I would just directly render it at line 99, and 104

Reply:
Good idea.